src/components/Sidebar/Sidebar.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import classnames from 'classnames';
 
-const Sidebar = ({ children, className, side = 'left', sticky = false }) => (
+const Sidebar = ({
+  children,
+  className,
+  side = 'left',
+  sticky = false,
+}) => (
   <aside
     className={
-      classnames('md:w-72 border-gray-100', "overflow-y-scroll", {
+      classnames(className, 'border-gray-100 overflow-y-scroll', {
```

Comment:
Moved the width setting out of this component so that width can be set by the parent.